src/event-handler.js

```diff
@@ -16,7 +16,10 @@ A non-ecommerce event has the following schema:
 function EventHandler(common) {
     this.common = common || {};
 }
-EventHandler.prototype.logEvent = function(event) {};
+EventHandler.prototype.logEvent = function(event) {
+    if (event.EventName == "click" || event.EventName == "change" || event.EventName == "submit") return;
+    window.heap.track(event.EventName, event.EventAttributes);
+};
```

Comment:
I think having too many OR conditions makes this a little harder to read.

I personally would use an array of `ignoredEvents` to make the code more self-documenting. Also, this way it makes it easier to add/remove an event in the future.

```
    var ignoredEvents = [
      'click',
      'change',
      'submit',
    ];

    if (ignoredEvents.includes(event.EventName.toLowerCase()) {
        return;
    }
    window.heap.track(event.EventName, event.EventAttributes);
```

Given that it's only 3 events right now, I'm fine with keeping it as is, but I would recommend using `event.EventName.toLowerCase()` in case an event is ever sent in `Capitalized` or `ALL_CAPS`.

If you don't go for the array approach, would consider turning the event names into constants so you can avoid spelling mistakes in the future.

For example:
```
var CLICK = 'click';
var CHANGE = 'change;
var SUBMIT = 'submit';

if (event.EventName === CLICK || event.EventName === CHANGE || event.EventName === SUBMIT) return;
```

Reply:
Took your advice and moved to using an array.